roles/odf_setup/defaults/main.yml

```diff
@@ -30,3 +30,6 @@ default_storageclass_annotation: '{"storageclass.kubernetes.io/is-default-class"
 
 # OCS gatherer_image
 gatherer_image: registry.access.redhat.com/ubi8/ubi
+
+# OpenShift Command Line Interface path
+odf_oc_tool_path: "{{ oc_tool_path | default('/usr/loca/bin/oc') }}"
```

Comment:
Let's deprecate the use of `oc_tool_path` and instead use the role variable i.e. `<role_prefix>_oc_tool_path`

Reply:
the default path is also wrong s/loca/local/